ls1intum/Artemis
# add further navigation bar improvement suggestions

### Checklist

### Description

show different exam participation status in the exam exercise navigation bar
the different status:
- synced
- not synced
- submitted
### Steps for Testing

- check the navigation bar button turns yellow when exercise participation is not synced
-check the icon of the exercise button changes when exercise participation is submitted

### Screenshots


![image](https://user-images.githubusercontent.com/50635005/85857722-aea3ec80-b7ba-11ea-9ec5-51de32e5ffe2.png)
![image](https://user-images.githubusercontent.com/50635005/85859661-ce88df80-b7bd-11ea-9fa3-ef0b3f384bc1.png)
